cmd/goal/asset.go

```diff
@@ -110,7 +110,7 @@ func init() {
 	addTxnFlags(freezeAssetCmd)
 
 	infoAssetCmd.Flags().Uint64Var(&assetID, "assetid", 0, "ID of the asset to look up")
-	infoAssetCmd.Flags().StringVar(&assetUnitName, "asset", "", "Unit name of the asset to look up")
+	infoAssetCmd.Flags().StringVar(&assetUnitName, "unit", "", "Unit name of the asset to look up")
```

Comment:
unfortunately, it's more than just a doc change, as suggested by the PR title, and it will break the goal backward compatibility.
( not to say it's not a good change. I think it's ok, but we need to make sure we communicate this one clearly )

Reply:
oops, I only meant to include the doc change. I'll get this part removed, it is part of a different PR.